config/locales/en.yml

```diff
@@ -257,6 +257,9 @@ en:
               reused:
                 one: "has been used before. Please choose one that is different from your last one."
                 other: "has been used before. Please choose one that is different from your last %{count}."
+              match:
+                confirm: "Confirm new password."
+                description: "Password should match with the one you have typed in New password field."
```

Comment:
Not a native speaker myself but I'd prefer: "'Password confirmation' should match the input in the 'New password' field."